### DISCUSSION TITLE
Right-click menu problem registered with clangd

### DISCUSSION THREAD

**Discussion by zx616456607** [clangd/clangd#1964] [Q&A] [answered]
![image](https://github.com/clangd/clangd/assets/19658190/66a636d0-c9a4-4192-bcc4-e237018b5969)

Are these menus registered with clangd?

**Comment by HighCommander4** [comment, marked as answer]
They are implemented by clangd, if that's what you mean.

**Reply by zx616456607** [reply]
Clangd 16.0.0.
Vscode-clangd 0.1.24.
Theia 1.41.0.

My environment version is the same as above, but without a few buttons in the image above, why?

**Reply by zx616456607** [reply]
I found the reason is that I need to install the plug-in Reference Search View
